index.bs

```diff
@@ -2729,6 +2769,29 @@ The steps for [=issuing a credential request to an authenticator=] are as follow
 
     1. Return [TRUE].
 
+#### Authentication API Exceptions #### {#sctn-authentication-api-exceptions}
```

Comment:
ditto

Reply:
Addressed in https://github.com/w3c/webauthn/pull/2047/commits/6760c571dc51670d256ddbc37fd4ddf8dda8b8c0 ✌️